### ISSUE TITLE
How to transfer Elementor Customizer values to JS file to configure Swiper Slider?

### ISSUE THREAD

**Issue by UrchenkoV** [elementor/elementor#9243]
How to transfer Elementor Customizer values to JS file to configure Swiper Slider?

Looked through half of the Internet and clear instructions have not found.
Please write step by step how to do it?

Thanks.

**Comment by UrchenkoV** [comment]
@black-eye 

To connect Swiper to a widget add this code to your widget
```
public function get_script_depends() {
      return [ 'swiper' ];
}
```
Here is a complete example where you need to register it
```
	public function get_name() {
		return 'YOUR';
	}

	public function get_title() {
		return __( 'YOUR', 'SLUG' );
	}

	public function get_icon() {
		return 'YOUR';
	}

	public function get_categories() {
		return [ 'YOUR' ];
	}

	public function get_script_depends() {
                     return [ 'swiper' ];
    }
```

**Comment by black-eye** [comment]
@UrchenkoV Thanks.
And will this code connect the the setings like slides per page, navigation or autoplay?

**Comment by UrchenkoV** [comment]
@black-eye 

No. This is just a Swiper connection, use the instructions to set up the Sweeper https://swiperjs.com/api/

**Comment by black-eye** [comment]
@UrchenkoV 
Well yes, I've actually came through this process and made it work (even in editor, which was a bit of challenge), but I was actually expecting some kind of prepared solution. The native carousel/slider widgets don't do it the way I've done it, it's somehow incorporated to the core and I've just haven't found a way, how it's done. But I wouldn't call myself a great programmer, so maybe I'm missing something obvious. And it worth mentioning, it was some time ago, before the JS core changes in 2.7.

**Comment by UrchenkoV** [comment]
@black-eye 

The about than you write I himself want to know so wrote this question.

**Comment by astbarc** [comment]
I'd also like to know how to access elementor Swiper to modify the settings and implement lazy load.

Thanks.

**Comment by black-eye** [comment]
@UrchenkoV That's what I've though, and that's why I've added my +1 to you request 😃
